src/lint/lint.test.ts

```diff
@@ -126,3 +136,31 @@ describe("#validateCustom", () => {
     stub.restore();
   });
 });
+
+describe("#validateModel", () => {
+  let testModel: model.document.Document;
+
+  beforeEach(async () => {
+    await AMF.init();
```

Comment:
Doesn't this only need to be done once, rather than before every test?

Reply:
Perhaps.  But I didn't want to change this as part of this PR.